aiohttp/websocket.py

```diff
@@ -172,6 +173,7 @@ def WebSocketParser(out, buf):
                     Message(OPCODE_BINARY, data, ''), len(data))
 
 
+native_byteorder = sys.byteorder
```

Comment:
`sys.byteorder` is little (at least for me), while iirc websocket uses network one which is big. Could anyone confirm this?